test/videojs-contrib-hls.test.js

```diff
@@ -1786,7 +1787,7 @@ QUnit.test('uses mobile default bandwidth if browser is Android', function(asser
   openMediaSource(this.player, this.clock);
 
   assert.equal(this.player.tech_.hls.bandwidth,
-               500000,
+               4194304,
```

Comment:
We discussed that we were going to use the desktop bandwidth number everywhere, so this test had to be updated to account for that (I figured I would update over removing).